packages/connector/implementation/src/integrations/dbt/BigQueryDbtIntegration.kt

```diff
@@ -0,0 +1,39 @@
+package io.tellery.integrations
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.module.kotlin.readValue
+import entities.NewProfile
+import io.tellery.connectors.BigQueryConnector
+import io.tellery.connectors.fields.BigQueryFields
+import java.util.*
+
+@Type("BigQuery")
+class BigQueryDbtIntegration : DbtIntegration() {
+
+    override fun transformToDbtProfile(profile: NewProfile): BigQueryDbtProfile {
+        profile.let {
+            val jsonBody = Base64
+                .getDecoder()
+                .decode(getValueOrThrowException(it, BigQueryFields.KEY_FILE))
+                .decodeToString()
+
+            val keyfileJson: BigQueryConnector.BigQueryKeyBody = mapper.readValue(jsonBody)
+
+            return BigQueryDbtProfile(
+                method = "service-account-json",
+                priority = "interactive",
+                dataset = getValueOrThrowException(it, BigQueryFields.DATASET),
+                keyfileJson = keyfileJson,
+                project = keyfileJson.projectId
+            )
+        }
```

Comment:
```suggestion
        val jsonBody = Base64
            .getDecoder()
            .decode(getValueOrThrowException(profile, BigQueryFields.KEY_FILE))
            .decodeToString()

        val keyfileJson: BigQueryConnector.BigQueryKeyBody = mapper.readValue(jsonBody)

        return BigQueryDbtProfile(
            method = "service-account-json",
            priority = "interactive",
            dataset = getValueOrThrowException(it, BigQueryFields.DATASET),
            keyfileJson = keyfileJson,
            project = keyfileJson.projectId
        )
```
meaningless let here?

Reply:
It was used in two places.